public/locales/fr/landing.json

```diff
@@ -79,25 +82,37 @@
     },
     "text": "Sur un reçu officiel où le numéro de référence est encerclé, vous trouverez\u00a0:"
   },
-  "receipt-explanation": "Le numéro de dossier ou le numéro de référence sert à identifier votre demande de passeport. Le numéro de dossier ou le numéro de référence figure sur le reçu qui vous a été remis si vous avez déposé votre demande de passeport en personne. Si vous avez déposé votre demande par la poste, ou si vous n'avez pas de numéro de dossier ou de numéro de référence, <Link>vous pouvez demander à le recevoir</Link>.",
   "receipt-image-1": {
     "alt": "Un exemple de reçu officiel. Le numéro de dossier est entouré au centre du reçu.",
     "descriptive-text": "<strong>Exemple de reçu officiel. Le numéro de dossier est encerclé au centre du reçu.</strong>",
     "src": "/Receipt1_FR.png",
-    "title": "Exemple de reçu avec un numéro de dossier 1"
+    "title": "Example de reçu 1"
   },
   "receipt-image-2": {
     "alt": "Exemple d'un reçu officiel et bordereau de cueillette. Le numéro de dossier est encerclé en haut à gauche du reçu, au-dessus d'un code-barres.",
     "descriptive-text": "<strong>Exemple d'un reçu officiel et bordereau de cueillette. Le numéro de dossier est encerclé en haut à gauche du reçu, au-dessus d'un code-barres.</strong>",
     "src": "/Receipt2_FR.png",
-    "title": "Exemple de reçu avec un numéro de dossier 2"
+    "title": "Example de reçu 2"
```

Comment:
Was this intentional? It is spelled `exemple` in french, isn't it?
(see line 93 above)